src/config/options.js

```diff
@@ -116,6 +99,8 @@ export const options = {
       }
     ]
   },
+  // TODO: This will be cleaned up once the following PR is merged
+  // https://github.com/AlexsLemonade/refinebio-web/pull/365
```

Comment:
As noted here, `transformation` will be removed once #365 is merged (it has already been removed in that PR).